tonic-build/src/lib.rs

```diff
@@ -92,7 +92,6 @@ pub fn fmt(out_dir: &str) {
             .output()
             .unwrap();
 
-        println!("out: {:?}", out);
```

Comment:
I think this was originally here to give you the context on the error, maybe it just makes sense to actually destructure this error and print something nice when it fails so the user can figure out what went wrong, what do you think?